src/main/java/org/broadinstitute/dsm/model/filter/participant/BaseFilterParticipantList.java

```diff
@@ -106,9 +109,13 @@ public BaseFilterParticipantList(String jsonBody) {
             }
 
             logger.info("Found query conditions for " + mergeConditions.size() + " tables");
+            GoogleAnalyticsMetricsTracker.getInstance().sendAnalyticsMetrics(ddpInstance.getName(), GoogleAnalyticsMetrics.EVENT_CATEGORY_PARTICIPANT_LIST,
+                    GoogleAnalyticsMetrics.EVENT_ACTION_PARTICIPANT_LIST, GoogleAnalyticsMetrics.EVENT_LABEL_PARTICIPANT_LIST,  Math.toIntExact((System.currentTimeMillis() - timer)/1000));
```

Comment:
extracting this code `Math.toIntExact((System.currentTimeMillis() - timer)/1000)` as method with meaningful name would be useful for quick understanding in the future.